src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -501,85 +676,315 @@
   &-year-partial,
   &-month-partial {
     .picker-cell-inner {
-      width: 56px;
+      width: 88px;
+    }
+  }
+
+  &-large {
+    .picker-year-partial,
+    .picker-month-partial {
+      .picker-cell-inner {
+        width: 104px;
```

Comment:
magic numbers throughout.. consider making the scss at a minimum and then they are self documenting

Reply:
Same, with 4px adjustment for padding

Reply:
Same. The month and year picker cells are novel. They are not circles.

Reply:
Done